eslint.config.js

```diff
@@ -0,0 +1,134 @@
+/*
```

Comment:
This is the new flat config file format will support from v9. Plugins need to change their code in order to work with v9. Right now I use a compatibility layer in order to keep everything still going.

Eslint v9 was released a few days ago. Even with the compatibility layer, some plugins (import or react) have rules that are breaking. I will stick for the moment with v8.57.0. When the plugins will be updated, we can upgrade to Eslint v9, and the migration should go faster since we already use the new config file.